### DISCUSSION TITLE
Helm chart repo url

### DISCUSSION THREAD

**Discussion by ravanapel** [aquasecurity/starboard#270] [Questions & Help] [answered]
In the [documentation](https://github.com/aquasecurity/starboard#with-helm) there is instructions on how to install the starboard operator with helm. However, I am unable to find the repo URL. Can you point me to the URL of the chart repository?

**Comment by danielpacak** [comment, marked as answer]
👋 @ravanapel We haven't published the Helm chart to any repo yet. You have to clone the Git repository and install the Helm chart with the local filesystem path reference.

We started thinking about hosting our own Helm repo as GitHub pages so stay tuned!
